index.js

```diff
@@ -52,7 +53,13 @@ class Geocoder {
   }
 
   end (cb) {
-    const req = http.get(this.httpOptions, responseHandler.bind(this, cb))
+    // I put http here since you're using the http module.
+    const req = http.get('http://' +
```

Comment:
This could cause an error if the user adds an url to the options that already starts with 'http://', we could just add an if statement in the constructor to slice off the 'http://'. Adding to that I don't know how this works out when the User wants to use https (for which we could add another option).